### PR TITLE
Fix compiler override warnings

### PR DESCRIPTION
**Pull request by konopka**

### CODE REVIEW

File: cocos/editor-support/spine/SkeletonRenderer.h [hunk 1]
```diff
@@ -84,10 +84,10 @@ class SkeletonRenderer: public cocos2d::Node, public cocos2d::BlendProtocol {
...
 
 	// --- BlendProtocol
-	virtual void setBlendFunc (const cocos2d::BlendFunc& blendFunc);
-	virtual const cocos2d::BlendFunc& getBlendFunc () const;
-	virtual void setOpacityModifyRGB (bool value);
-	virtual bool isOpacityModifyRGB () const;
+	virtual void setBlendFunc (const cocos2d::BlendFunc& blendFunc) override;
+	virtual const cocos2d::BlendFunc& getBlendFunc () const override;
+	virtual void setOpacityModifyRGB (bool value) override;
+	virtual bool isOpacityModifyRGB () const override;
```
Comment (minggo) [inline cocos/editor-support/spine/SkeletonRenderer.h]: According to Spine runtime license, we can not modify it.

File: cocos/editor-support/cocostudio/CCArmatureAnimation.h [hunk 2]
```diff
@@ -173,7 +173,7 @@ class  CC_STUDIO_DLL ArmatureAnimation : public ProcessBase
      */
     ssize_t getMovementCount() const;
 
-    void update(float dt);
+    void update(float dt) override;
```
Comment (minggo) [inline cocos/editor-support/cocostudio/CCArmatureAnimation.h]: should add `virtual`?

File: cocos/3d/CCSprite3D.h [hunk 3]
```diff
@@ -165,7 +165,7 @@ class CC_DLL Sprite3D : public Node, public BlendProtocol
     Sprite3D();
     virtual ~Sprite3D();
     
-    bool init();
+    bool init() override;
```
Comment (minggo) [inline cocos/3d/CCSprite3D.h]: should add `virtual`?

**Comment by minggo** [comment]
Thanks @konopka .
On which platform you found this warnings?

**Comment by konopka** [comment]
Ubuntu with clang compiler

**Comment by minggo** [comment]
Thanks @konopka 
Could you please send a pull request to v4-develop too?

**Pull request merged**